lib/ramble/ramble/application.py

```diff
@@ -1824,11 +1825,17 @@ def calculate_statistics(self, workspace):
         namespace.
         """
 
-        def is_numeric(value):
-            """Returns true if a fom value is numeric"""
+        def is_numeric_fom(fom):
```

Comment:
Could it make sense to move this into the FomType class? I'm not sure if it makes the most sense yet...

Maybe at a minimum put a TODO to help us remember to move it once we create a Fom class.

Reply:
At first glance I don't find it intuitive, because the FomType class is an enum that doesn't have any information about the FOM inside of it. I think a FOM class seems like a better fit. Added todo.